.github/workflows/hybrid-integration-test.yaml

```
@@ -0,0 +1,30 @@
+name: integration test
+
+on:
+  push:
```

please add the paths, so this is run only when pushing hybrid attestor code
```suggestion
  push:
    paths:
    - 'hybrid/**'
```